client/packages/core/src/Reactor.js

```diff
@@ -1354,6 +1354,14 @@ export default class Reactor {
     };
   }
 
+  async getAuth() {
+    const {user, error} = await this.getCurrentUser();
+    if (error) {
+      throw new Error(error);
```

Comment:
nit: If `error` is an object, I am not sure that you can throw it this way. Would try to test this case and make sure